src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapExpungeScenario.scala

```diff
@@ -0,0 +1,59 @@
+package com.linagora.gatling.imap.scenario
+
+import java.util.Calendar
+
+import com.linagora.gatling.imap.PreDef._
+import com.linagora.gatling.imap.protocol.command.FetchAttributes.AttributeList
+import com.linagora.gatling.imap.protocol.command.MessageRange._
+import com.linagora.gatling.imap.protocol.command.MessageRanges
+import com.linagora.gatling.imap.protocol.command.StoreFlags
+import io.gatling.core.Predef._
+import io.gatling.core.scenario.Simulation
+import io.gatling.core.session.Expression
+
+import scala.collection.immutable.Seq
+import scala.concurrent.duration._
+import scala.util.Random
+
+class ImapExpungeScenario extends Simulation {
+  val numOfMailInInbox = 200
+  val percentageOfMailToExpunge = 20.0
+
+  val userFeeder = csv("users.csv").circular
+  val userCount: Int = userFeeder.records.length
+
+  def getRandomDeleted: Boolean = Random.nextFloat() < (percentageOfMailToExpunge/ 100.0)
+
+  def flagsWithRandomDeletion = (session: Session) => {
+    session.get("username").validate[String].map( _ =>
```

Comment:
session.get('username') is here only to make the scala compiler happy, it's not necessary and ugly because we should not fail even iff there is no user.
I am open to any suggestion to fix that.